FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -603,6 +656,70 @@ await userManager.DeleteUserAsync(uid, cancellationToken)
                 .ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Deletes the users specified by the given identifiers.
+        /// <para>
+        /// Deleting a non-existing user won't generate an error. (i.e. this method is idempotent.)
+        /// Non-existing users will be considered to be successfully deleted, and will therefore be
+        /// counted in the DeleteUserResult.SuccessCount value.
```

Comment:
Suggest backticks for DeleteUserResult.SuccessCount, here and similar copies.

Reply:
Done.